tls/s2n_prf.c

```diff
@@ -372,7 +380,28 @@ int s2n_prf_master_secret(struct s2n_connection *conn, struct s2n_blob *premaste
     label.data = master_secret_label;
     label.size = sizeof(master_secret_label) - 1;
 
-    return s2n_prf(conn, premaster_secret, &label, &client_random, &server_random, &master_secret);
+    return s2n_prf(conn, premaster_secret, &label, &client_random, &server_random, NULL, &master_secret);
+}
+
+int s2n_hybrid_prf_master_secret(struct s2n_connection *conn, struct s2n_blob *premaster_secret)
+{
+    struct s2n_blob client_random, server_random, master_secret;
```

Comment:
Style wise I would combine the declaration of these blobs with their initialization of .data and .size.